src/Domain/CreateInitialElement.cpp

```diff
@@ -52,6 +52,9 @@ Element<VolumeDim> create_initial_element(
     // the neighbor if necessary.
     auto neighbor_is_refined = make_array<VolumeDim>(false);
     for (size_t d = 0; d < VolumeDim; ++d) {
+      if (d == direction_in_neighbor.dimension()) {
```

Comment:
Could you add a comment explaining why you don't need to do anything in this perpendicular direction? I think you're just not using the output of this function for perpendicular directions in the code below, but I'm not quite sure at this point.

Reply:
A code comment sounds good, but I don't understand the second part of what you wrote.  Would "The refinement difference perpendicular to the interface is not restricted." make things clear?

Reply:
yes sounds good, thanks